### PR TITLE
Allow to use Carrierwave v0.11.0

### PR DESCRIPTION
**Pull request by SergeyKishenin**

### CODE REVIEW

File: carrierwave-mongoid.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "carrierwave", [">= 0.8.0", "< 0.11.0"]
+  s.add_dependency "carrierwave", [">= 0.8.0", "<= 0.11.0"]
```
Comment (rmm5t) [inline carrierwave-mongoid.gemspec]: This needs to be a `< 0.12.0`

**Comment by rmm5t** [comment]
Thanks. Please make the change addressed in the comment, and I'll merge this in.

However, please note that we probably won't release an official new version of this gem until #150 is also resolved.

**Comment by SergeyKishenin** [comment]
Done

**Comment by rmm5t** [comment]
Thanks! :beers:

**Comment by rmm5t** [comment]
* carrierwave-mongoid v0.9.0 is now officially released with carrierwave v0.11 support.
* Note: This release does not yet include `mount_uploaders` support. Waiting on a viable PR for that.

**Comment by veeruwithrails** [comment]
which version of carrierwave-mongoid supports ` mount_uploaders`

**Comment by rmm5t** [comment]
@veeruwithrails Are you trolling me? :trollface:

> Note: This release does not yet include mount_uploaders support. Waiting on a viable PR for that.

**Pull request merged**